language/id.yml

```diff
@@ -1,149 +1,439 @@
+no-mute-user: Berikan saya pengguna yang akan dibisukan!
+no-unmute-user: Berikan saya pengguna yang akan tidak dibisukan dan membuat mereka bisa berbicara kembali!
+mute-success: Berhasil dibisukan **{}**!
+mute-success-time: Berhasil dibisukan **{}** untuk {}!
+unmute-done: Sudah tidak dibisukan!\n pengguna ini bisa berbicara kembali!
+cant-mute-admin: Saya tidak akan membisukan seorang Administrator!
+self-muting: Hahaha, saya tidak akan membisukan diri saya sendiri!
+already-muted: Pengguna ini sudah dibisukan!
+user-not-muted: Pengguna ini tidak dibisukkan dan memiliki ijin untuk bisa berbicara!
+#endregion
+#region note
+notes-button: Catatan
+notes-help: |
+  \nSimpan data yang akan dibaca atau digunakan dikemudian hari didalam catatan!\n
+  **Perintah pengguna:**
+  × /get <nama catatan>: Menampilkan catatan dengan nama catatan ini. Contoh, jika anda ingin membuka catatan bernama test, itu akan menjadi /get test.
+  × #<nama catatan>: Sama seperti /get. Contoh : #test
+  × /notes: Daftar semua catatan pada grup ini.\n
+  **Perintah Administrator:**
+  × /save <nama catatan> <isi catatan>: menyimpan catatan baru dengan nama "<nama catatan>". Dengan cara membalas pesan yang akan disimpan pesannya.
+  × /clear <nama catatan> | /delnote <nama catatan>: Hapus catatan yang tersimpan jika ada dan tersimpan.
+  \nCek /markdownhelp untuk info format catatan lainnya silahkan dicek.
+notes-invalid-args : |
+ Membalas pesan yang akan disimpan pesannya atau memberikan isi setelah nama catatan.
+  Contoh: /save <nama catatan> <isi catatan/balas ke pesan>
+note-saved: "Pesan disimpan dengan nama `{}`."
+note-list: "**Daftar semua catatan pada grup ini {}:**\n"
+no-notes: "`Tidak ada catatan pada grup ini!`"
+notes-del-noargs: Berikan saya nama catatan yang akan dihapus!
+notes-not-exist: Tidak ada catatan itu pada grup ini!
+notes-deleted: "Catatan `{}` dihapus!."
+#endregion
 #region purge
 purges-button: Pembersihan
 purges-help: |
-  Hapus pesan obrolan dengan mudah dengan perintah ini.\n
-  ** Hanya Admin: **
+  Hapus pesan di grup dengan mudah menggunakan saya dengan perintah berikut ini.\n
+  ** Hanya Administrator yang bisa menggunakan: **
   × /del: Hapus pesan yang dibalas.
-  × /purge | /prune: Menghapus semua pesan hingga pesan dibalas.
-purge-done: "`{} pesan dibershikan dalam {} detik...`"
-purge-error: "__Tidak bisa membersihkan pesan lebih dari 2 hari__"
+  × /purge | /prune: Menghapus semua pesan hingga pesan yang dibalas /purge.
+purge-done: "`{} pesan telah dihapus dalam {} detik...`"
+purge-error: "__Mohon maaf tidak bisa menghapus pesan lebih dari 2 hari__"
+#endregion
+#region reporting
+reporting-button: Laporan
+reporting-help: |
+  **Hanya Administrator yang bisa menggunakan:**
+  × /reports <yes/no/on/off>: aktifkan atau nonaktifkan laporan grup.
+    - jika dijalankan di grup akan merubah pengaturan grup dan akan menampilkan notifikasi.
+    - jika dijalankan di PM akan merubah pengaturan kamu dan akan menampilkan notifikasi.
```

Comment:
refer to https://github.com/userbotindo/Anjani/pull/32#discussion_r713908400 [merubah -> mengubah].
There are some of this words on `reporting-help`, `greetings-help`, `clean-serv-set`, `cust-welcome-set`, `reset-welcome`, `cust-goodbye-set`, `reset-goodbye`

Reply:
Let me check again. Sorry.